src/libraries/Fuzzing/DotnetFuzzing/Program.cs

```diff
@@ -361,7 +384,14 @@ private static string GenerateLocalRunHelperScript(IFuzzer fuzzer)
     {
         string script = $"%~dp0/libfuzzer-dotnet.exe --target_path=%~dp0/DotnetFuzzing.exe --target_arg={fuzzer.Name}";
 
-        if (fuzzer.Dictionary is not null)
+        // We don't support dictionaries and corpora at the same time yet, and some fuzzers
```

Comment:
Nit: Any reason why not?
The corpus will be ignored by OneFuzz, but I don't see why we should block it locally.

Reply:
This changes only the local run script, OneFuzz runs are unaffected.

Since we don't have corpora setup for OneFuzz (yet), some fuzzers place example inputs as dictionary entries instead, but that does not work as well as having them in the corpus. I didn't want to remove the dictionaries entirely because that might slow down OneFuzz runs, so as a temporary compromise, corpus takes precedence over dictionary when running locally (as it is more efficient to omit the suboptimal dictionary in this case).

Of course, having both at the same time (corpus of example whole inputs, dictionary with the right "alphabet" from which to compose inputs) would be best and is something we should aim for the future.